### PR TITLE
Tweak navbar and sidebar styles to handle smaller viewports

### PR DESCRIPTION
**Pull request by scottopherson**
Until a more permanent solution is implemented, this commit retrofits the main navbar and sidebar styles to better handle smaller viewport sizes. The bulk of this diff is tweaks to paddings and font-sizes within Carbon grid breakpoints. The largest visual change is the main navbar collapsing to two rows within smaller viewports.

<details>
  <summary>Before</summary>
  
</details>

<details>
  <summary>After</summary>
  
</details>

### CODE REVIEW

File: components/automate-ui/src/app/page-components/navbar/navbar.component.html [hunk 1]
```diff
@@ -1,55 +1,53 @@
 <header role="banner">
   <nav role="navigation" class="navigation-wrapper">
```
Comment (scottopherson) [inline components/automate-ui/src/app/page-components/navbar/navbar.component.html:2]: No changes to this html other than removing the containing `div.left-nav`.

**Comment by scottopherson** [comment]
@susanev oh good catch, I'll whip up a fix.

**Comment by srenatus** [comment]
This is great! I can't assess the change itself, but this has been annoying me lately, a bit -- when opening dev tools in on-the-side mode. :clap:

**Pull request merged**